src/style.css

```
@@ -0,0 +1,76 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  width: 100vw;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  height: 100%;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: nowrap;
+  list-style-type: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  padding: 0;
+  margin: 0;
+  line-height: 60px;
+  margin-left: -20px;
+}
+
+.nav-list__item {
+  margin-left: 20px;
+}
+
+.nav-list__item__link {
+  display: block;
+  position: relative;
+  text-decoration: none;
+  color: #000;
+}
+
+.nav-list__item__link::after {
+  content: "";
+  position: absolute;
+  height: 4px;
+  width: 100%;
+  background-color: #00acdc;
+  left: 0;
+  bottom: 0;
+  border-radius: 8px;
+  visibility: hidden;
+}
+
+.nav-list__item__link:hover {
+  color: #00acdc;
+}
+
+.nav-list__item__link.is-active {
+  color: #00acdc;
+}
```

```suggestion
.nav-list__item__link:hover,
.is-active {
  color: #00acdc;
}
```
Don't repeat yourself